packages/@vue/cli/lib/add.js

```diff
@@ -27,12 +26,7 @@ async function add (pluginName, options = {}, context = process.cwd()) {
   log(`${chalk.green('✔')}  Successfully installed plugin: ${chalk.cyan(packageName)}`)
   log()
 
-  const generatorPath = resolveModule(`${packageName}/generator`, context)
-  if (generatorPath) {
-    invoke(pluginName, options, context)
-  } else {
-    log(`Plugin ${packageName} does not have a generator to invoke`)
```

Comment:
Why remove this?

Reply:
This is just an useless duplicate. `invoke` already does the same check at https://github.com/pksunkara/vue-cli/blob/hook/packages/%40vue/cli/lib/invoke.js#L64 pretty early.

Reply:
Ah that's true, thanks.